src/components/Dropdown/Dropdown.tsx

```diff
@@ -349,10 +364,32 @@ const Dropdown = ({
     }
 
     setIsOpen(true);
+
+    window.setTimeout(() => {
+      const focusedElement = document.activeElement;
+
+      if (focusedElement && focusedElement.id === `${name}-dropdown-button`) {
+        const button = focusedElement.parentNode as HTMLElement | undefined;
+        const optionsContainer = button ? button.nextElementSibling : null;
```

Comment:
can we not use `isOpen` for the conditional and the button/optionsContainer refs for the variables here?

Reply:
Hmm I'll look into it -- that's just what was in place [beforehand](https://github.com/Headstorm/foundry-ui/blob/master/src/components/Dropdown/Dropdown.tsx#L421-L424)